### ISSUE TITLE
Problem loading multiple Guassian logfiles in python 3.4

### ISSUE THREAD

**Issue by ChayaSt** [cclib/cclib#219]
I am currently using cclib version 1.3.2. While I have no issues loading multiple files into the Gaussian parser when using python 2.4, when I try to do the same in python 3.4 I get the following error:

```
<ipython-input-8-e2685fd24107> in <module>()
----> 1 test.parse()

/Users/sternc1/anaconda/envs/py3k/lib/python3.4/site-packages/cclib/parser/logfileparser.py in parse(self, progress, fupdate, cupdate)
    246             # Any attributes can be freely set and used across calls, however only those
    247             #   in data._attrlist will be moved to final data object that is returned.
--> 248             self.extract(inputfile, line)
    249 
    250         # Close input file object.

/Users/sternc1/anaconda/envs/py3k/lib/python3.4/site-packages/cclib/parser/gaussianparser.py in extract(self, inputfile, line)
    141             self.updateprogress(inputfile, "Symbolic Z-matrix", self.fupdate)
    142 
--> 143             line = inputfile.next()
    144             while line.split()[0] == 'Charge':
    145 

AttributeError: 'FileInput' object has no attribute 'next'
```

I use a list of path names to load the files. 
```
logfiles = glob.glob('testing/reference/PRL.scan2.*')
scan = Gaussian(logfiles)
data = scan.parse()
```
Please advise.
Thanks

**Comment by langner** [comment]
This should actually be fixed by pull request #218...

**Comment by langner** [comment]
I merged in #218 and confirmed multiple files work for Python3.4. Please use the master branch.

**Comment by ChayaSt** [comment]
Thanks! It's working.
